mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/data-migration/set_administrative_notes.sql

```diff
@@ -0,0 +1,13 @@
+-- mapping_profiles:
+-- change values: set administrative notes for mapping profiles, which haven't one
+UPDATE ${myuniversity}_${mymodule}.mapping_profiles
+SET jsonb = jsonb_insert(jsonb, '{mappingDetails,mappingFields,0}', '{"name": "administrativeNotes", "path": "instance.administrativeNotes[]", "value": "", "enabled": "true", "subfields": []}')
+WHERE (jsonb ->> 'mappingDetails')::json ->> 'recordType' = 'INSTANCE' AND (jsonb ->> 'mappingDetails')::json ->> 'mappingFields' NOT LIKE '%"name": "administrativeNotes"%';
```

Comment:
```suggestion
WHERE jsonb -> 'mappingDetails' ->> 'recordType' = 'INSTANCE' AND jsonb -> 'mappingDetails' ->> 'mappingFields' NOT LIKE '%"name": "administrativeNotes"%';
```
we can omit cast syntax